test_sound.js

```diff
@@ -1,19 +1,49 @@
 function makesound(){
+  var slide = document.getElementById("myRange");

```

Comment:
Please use `const` [when a variable doesn't need to be changed](https://google.github.io/styleguide/jsguide.html#features-use-const-and-let).

Also it's usually good practice to initialize these on a more global scope (i.e., outside of the function).